bom/pom.xml

```diff
@@ -666,62 +665,6 @@
             <artifactId>reactive-streams</artifactId>
             <version>${version.reactivestreams}</version>
          </dependency>
-         <dependency>
```

Comment:
The dependencies should still be defined, just excluding the version tag.

Reply:
Definitely not, it's a `DependencyManagement` not a `Dependencies` section.

Reply:
h0h0 my bad, so it is, I should have looked at the full diff :+1: